physionet-django/project/management/commands/get_metrics_data.py

```diff
@@ -0,0 +1,267 @@
+"""
+Command to:
+- Parse NGINX log files and write their metrics data to the database
+"""
+
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
+
+from project.models import CoreProject, PublishedProject, Metrics
+
+
+class Command(BaseCommand):
+    """
+    Command to parse a log file and load data to the Metrics table.
+    """
+    help = "Parses a log file and loads data to the Metrics table."
+
+    def add_arguments(self, parser):
+        parser.add_argument('files', nargs='+', type=str,
+                            help='Specify log file(s) to parse')
+        parser.add_argument('-c', '--is_cron', type=str_to_bool, default=False,
```

Comment:
`is_cron` is an interesting idea and perhaps something that we want, but as it is just used to check the date (as far as I can see) would it be clearer for this to be a `check_date` argument instead?

Reply:
Yes, I agree that `check_date` would be a clearer argument name, so I adjusted accordingly. Thank you!